### DISCUSSION TITLE
scss not compiling anymore

### DISCUSSION THREAD

**Discussion by eberlyjm** [bootscore/bootscore#736] [Q&A]
I recently used Bootscore for a project and it worked like a charm. Now for some reason the compiler does not seem to be working. If I add a new class to _bscore_custom (or change an existing attribute) and save, main.css does not update and the new class is not included in the main.css file. There is no error, but main.css looks like it did a week ago when I made the previous changes. It's as if the compiler suddenly stopped  working. I've restarted my computer and tried a number of different things (including disabling environment check in wp-config.php), but nothing. Tried renaming main.css, thinking maybe it would create a new compiled file, but it does not. 
The strange thing is that it used to work perfectly and nothing has changed on my system since then. 
I am using a child theme. Although, I tried the same thing in the parent theme, with same results (nothing is compiled).

**Comment by crftwrk** [comment]
Use developer mode to force compiler in your `wp-config.php` as described here https://bootscore.me/documentation/theme/bootstrap-css-sass/#Developer_mode

Solved?

**Comment by eberlyjm** [comment]
This did not change anything. But help me understand…
I though the compiling would happen on my computer when I save the _bscore_custom file, which would allow me to generate the main.css file and upload it to the server. I think maybe what actually happens is that the css is being compiled once the scss file is uploaded.
If so, that would explain why it was working previously, but not today (until just now). I was running a local version off my computer. So it was generating the main.css file locally. Once I quit localhost it was not compiling anymore locally.

**Reply by crftwrk** [reply]
Usually compiler starts if something in `_bscore_custom.scss` has changed. But it can happen that compiler stucks and does not get triggered. In that case you have to force the compiler:

- Use dev mode and add `define('WP_ENVIRONMENT_TYPE', 'development');` to your `wp-config.php`. Now scssphp recompiles on every page load, no matter if something has changed or not.
- Delete your child's compiled `main.csss`
- Move your project to a different server

We had this discussion already several times, please follow the pinned issue https://github.com/bootscore/bootscore/issues/228

**Reply by eberlyjm** [reply]
Thanks again for your help, Basti. I had seen #228 thread and had added the code to wp-config.php, but I think the problem was that had not understood that the change to the _bscore_custom.scss file has to occur on the REMOTE server. I was changing it locally and expecting my LOCAL main.css file to compile, but that does not happen locally unless I'm running my localhost server.

**Reply by crftwrk** [reply]
Simply delete compiled `main.css` in your child and reload the site. You will see that this file will be created and compiled again. This has nothing to do if you are on local or remote server.
